### PR TITLE
Visitor can navigate the site

### PR DESCRIPTION
**Pull request by pedbr**
-Creates landing page
-Creates Navbar
-Creates Feature tests

PT- https://www.pivotaltracker.com/story/show/167007911

### CODE REVIEW

File: app/views/products/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+      %li
+      %a{:href => "#"} Cart
+      %li
+      %a{:href => "#"} Login
+
+
+%h1 Menu
+
+
+
```
Comment (oliverochman) [inline app/views/products/index.html.haml]: Remove empty lines

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'products/index'
```
Comment (oliverochman) [inline config/routes.rb]: please remove this line because you don't need it, also old syntax

File: features/landing_page.feature [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Feature: See menu on landing page
+  As a visitor,
+  When I visit the landing page of the application
+  I would like to see a list of all dishes on menu
+
+  Scenario: View list of dishes on the landing page
+    When I visit the site
```
Comment (oliverochman) [inline features/landing_page.feature]: Needs to start with `Given`

File: app/views/products/index.html.haml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+%nav.navbar.navbar-default{:role => "navigation"}
+      %ul.nav.navbar-nav
+      %li
+      %a{:href => "#"} Cart
```
Comment (oliverochman) [inline app/views/products/index.html.haml]: We do not want to use hash rockets, old syntax

e.g
`%a{ href: "#" } Cart`

File: app/views/products/index.html.haml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+%nav.navbar.navbar-default{:role => "navigation"}
+      %ul.nav.navbar-nav
```
Comment (oliverochman) [inline app/views/products/index.html.haml]: make sure that you have the same tab size in all files

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: app/views/products/index.html.haml (hunk 5 above)
Comment (pedbr) [reply app/views/products/index.html.haml]: Corrected!

File: app/views/products/index.html.haml (hunk 4 above)
Comment (pedbr) [reply app/views/products/index.html.haml]: Done!

File: features/landing_page.feature (hunk 3 above)
Comment (pedbr) [reply features/landing_page.feature]: Corrected.

File: config/routes.rb (hunk 2 above)
Comment (pedbr) [reply config/routes.rb]: Removed.

File: app/views/products/index.html.haml (hunk 1 above)
Comment (pedbr) [reply app/views/products/index.html.haml]: Done!

File: features/step_definitions/basic_steps.rb [hunk 6]
```diff
@@ -0,0 +1,3 @@
+When("I visit the site") do
+    visit root_path
```
Comment (oliverochman) [inline features/step_definitions/basic_steps.rb]: fix tab size here

File: features/step_definitions/assertion_steps.rb [hunk 7]
```diff
@@ -0,0 +1,3 @@
+Then("I should see {string}") do |content|
+    expect(page).to have_content content
```
Comment (oliverochman) [inline features/step_definitions/assertion_steps.rb]: fix tab size here

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: features/step_definitions/basic_steps.rb (hunk 6 above)
Comment (pedbr) [reply features/step_definitions/basic_steps.rb]: Done!

File: features/step_definitions/assertion_steps.rb (hunk 7 above)
Comment (pedbr) [reply features/step_definitions/assertion_steps.rb]: Done!

**Pull request merged**